SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -48,6 +49,9 @@ class SPATIALGDK_API UAbstractLBStrategy : public UObject
 	virtual VirtualWorkerId WhoShouldHaveAuthority(const AActor& Actor) const
 		PURE_VIRTUAL(UAbstractLBStrategy::WhoShouldHaveAuthority, return SpatialConstants::INVALID_VIRTUAL_WORKER_ID;);
 
+	virtual SpatialGDK::FActorLoadBalancingGroupId GetActorGroupId(const AActor& Actor) const
```

Comment:
If so, I'd probably prefer to remove virtual altogether and leave a single implementation in the LayeredLBStrat, since it doesn't seem that we expect it to work with anything but the LayeredLBStrat.

Reply:
The only thing to look out for will be usages of DebugLBStrat since we won't be able to cast it to LayeredLBStrat :/

Reply:
I think it's better to leave this as is and fix this once we'll get to removing LB strats.